scripts/docs.sh

```diff
@@ -27,6 +27,7 @@
 #------------------------------------------------------------------------------
 
 set -e
+scripts/update_robots.sh
```

Comment:
I think the name is a bit confusing, would name it something like `update_robotstxt.sh`